bitnami/elasticsearch/values.yaml

```diff
@@ -742,6 +742,3 @@ kibana:
     hosts:
       - '{{ include "elasticsearch.coordinating.fullname" . }}'
     port: 9200
-  ## String to partially override elasticsearch.fullname template in the kibana subchart
-  ##
-  nameOverride: elasticsearch
```

Comment:
I guess you're just removing the default value but you can use this parameter, right?

Reply:
Yes, users can use it, but we don't need to set a default value